### ISSUE TITLE
Merging nearby bounding boxes using Tesseract

### ISSUE THREAD

**Issue by sophiajwchoi** [wenwenyu/PICK-pytorch#8]
Hi Wenwen,
In the training data set, I noticed a bounding box includes more than one word. Which program did you use to merge nearby bounding boxes based on distance? Currently, I have a bounding box around each word. Will the accuracy be affected if the bounding boxes are merged together? Thank you!

**Comment by wenwenyu** [comment]
@sophiajwchoi Bounding boxes are generated by the text detection algorithm. For example, EAST, PSENet, it directly predicts the coordinate of bounding boxes without merging word operation. Tesseract is a general detection method that may not suitable for task-specific detection. I recommend you to use the trained model on your datasets.

**Comment by tengerye** [comment]
Hi, @wenwenyu, when you refer "text detection", do you mean text line detection? Since text detection has many granularity, from character level to block level. Does your model apply to all levels?
